README.md

```diff
@@ -63,29 +63,29 @@ The latest pravega releases can be found on the [Github Release](https://github.
 
 ## Quick Start
 
-Read [Getting Started](docs/getting-started.md) page for more information, and also visit [sample-apps](https://github.com/pravega/pravega-samples) repo for more applications.
+Read [Getting Started](documentation/src/docs/getting-started.md) page for more information, and also visit [sample-apps](https://github.com/pravega/pravega-samples) repo for more applications.
 
 ## Running Pravega
 
-Pravega can be installed locally or in a distributed environment. The installation and deployment of pravega is covered in the [Running Pravega](docs/deployment/deployment.md) guide.
+Pravega can be installed locally or in a distributed environment. The installation and deployment of pravega is covered in the [Running Pravega](documentation/src/docs/deployment/deployment.md) guide.
 
 ## Support
 
-Don’t hesitate to ask! Contact the developers and community on the [mailing lists](https://groups.google.com/forum/#!forum/pravega-users) or on [slack](https://codecommunity.slack.com/#pravega) if you need any help. 
+Don’t hesitate to ask! Contact the developers and community on the [mailing lists](https://groups.google.com/forum/#!forum/pravega-users) or on [slack](https://codecommunity.slack.com/#pravega) if you need any help.
 Open an issue if you found a bug on [Github
 Issues](https://github.com/pravega/pravega/issues)
 
 ## Documentation
 
 The Pravega documentation of is hosted on the website:
-<http://pravega.io/docs> or in the [docs/](docs) directory of the source code.
+<http://pravega.io/docs/latest> or in the [documentation](documentation/src/docs) directory of the source code.
```

Comment:
http://pravega.io/docs/ exists but doesn't work on Internet Explore on Windows Phone, Chrome 65 on macOS Sierra, and Firefox 59 macOS Sierra.

I'll leave it up to your recommendation on whether:
- keep this as http://pravega.io/docs/latest and file an issue for http://pravega.io/docs
- for me to revert this to the broken http://pravega.io/docs and file an issue for http://pravega.io/docs